data/data-definitions.ts

```diff
@@ -11,10 +11,23 @@ export enum WeaponType {
   SNIPER = 'SNIPER',
   MARKSMAN = 'MARKSMAN',
   MELEE = 'MELEE',
+  LAUNCHER = 'LAUNCHER',
 }
 
 export interface CamoData {
-  [key: string]: number;
+  kill: number;
+  crouch?: number; // Kills while crouching
```

Comment:
basically `?:` means the field is optional
but `kill: number;` means each CamoData needs that field, otherwise vscode will throw error.

Here's what's cool. If you go to `data-definitions.ts`, highlight the `kill` field and press F2. If you rename it to something say hello, the fields in weapons-camo.ts will automatically get renamed.

Reply:
This is called refactoring and it's only possible with typed language (TypeScript vs JavaScript).